### PR TITLE
feat: Add stacks page with empty state

### PR DESCRIPTION
**Pull request by Diogomartf**
### Summary
Add Stacks page with empty state.

image

### CODE REVIEW

File: packages/app/app/stacks/empty.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import StackOfCoinsImage from "@/public/assets/images/empty-state-stacks-coins.svg";
+import { ButtonLink, HeadingText } from "@/ui";
+
+export const Empty = () => (
+  <div className="max-w-xl pt-32 mx-auto space-y-8">
+    <div className="flex flex-col items-center space-y-6 md:space-y-8">
+      <StackOfCoinsImage />
+      <div className="space-y-3 text-center">
+        <HeadingText size={2}>
+          {"Looks like you haven't created any stacks yet."}
```
Comment (berteotti) [inline packages/app/app/stacks/empty.tsx:10]: No need for this here, it can be simple text

File: packages/app/app/stacks/page.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+"use client";
+
+import { Empty } from "@/app/stacks/empty";
+
+export default function Page() {
+  return (
+    <div className="w-full max-w-screen-xl px-4 mx-auto">
+      <Empty />
```
Comment (berteotti) [inline packages/app/app/stacks/page.tsx:8]: `Empty` is too generic, maybe we can call it EmptyState or EmptyPage

**Review by berteotti — CHANGES_REQUESTED** [review]
Left some small comments. Also, did you optimised the SVG?

File: packages/app/app/stacks/empty.tsx (hunk 1 above)
Comment (Diogomartf) [reply packages/app/app/stacks/empty.tsx:10]: Typescript doesn't like all apostrophes ' .IMO it's easier like this than to use ampersand and stuff. But want to know your opinion.

**Comment by Diogomartf** [comment]
> Left some small comments. Also, did you optimised the SVG?

No. How should I do that?

File: packages/app/app/stacks/page.tsx (hunk 2 above)
Comment (Diogomartf) [reply packages/app/app/stacks/page.tsx:8]: I'll use `EmptyState`

**Comment by Diogomartf** [comment]
> Mobile menu is not linked

Yes, it's fixed now. Thanks

File: packages/app/components/navbar/MobileMenuList.tsx [hunk 3]
```diff
@@ -16,7 +16,7 @@ export default function MobileMenuList({ open, toggle }: MobileMenuListProps) {
         onClick={toggle}
       ></div>
       <div className="absolute left-0 w-full gap-2 px-6 py-2 border-b border-solid rounded-lg bg-surface-25 top-nav-height border-surface-75">
-        <Link href="#" className="flex items-center py-3 text-em-med">
+        <Link href="/stacks" className="flex items-center py-3 text-em-med">
```
Comment (berteotti) [inline packages/app/components/navbar/MobileMenuList.tsx:19]: Should we toggle the menu when user clicks on a new route?

Comment (Diogomartf) [reply packages/app/components/navbar/MobileMenuList.tsx:19]: Yes, we should

Comment (Diogomartf) [reply packages/app/components/navbar/MobileMenuList.tsx]: Fixed.

**Review by berteotti — DISMISSED** [review]
Looks good to me! Good job Diogo!

**Review by berteotti — APPROVED** [review]

**Pull request merged**